### PR TITLE
v1.24 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
 
+## 1.24.0 Release
+- Release Date: October 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
```
Comment (amyblais) [inline CHANGELOG.md]: Anything we want to highlight?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+ - Emoji picker search now ignores the leading colon.
+ - Added support for emoji not requiring a whitespace to render.
+ - Added support for footer and footer_icon in attachments.
+ - Added a password type for interactive dialogs.
+ - Added support for introductory markdown paragraph in interactive dialogs.
+ - Added support for boolean elements in interactive dialogs.
+ - Removed the full screen "Refresh" button when opening a team or channel with no cached data when offline.
+ - If Mattermost doesn't have permission to the photo library because the user denied the first prompt, the app now prompts to give permission to send the user to the settings to manually change the permission.
+
+### Bug Fixes
```
Comment (amyblais) [inline CHANGELOG.md]: List will be further updated in the coming week.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+ - Added sidebar and theming UI/UX improvements.
+ - Improved post menu animation and timing.
+ - Added a confirmation dialog when posting a message with `at-channel`.
+ - Added support for safe area in landscape view on iOS.
+ - Changed recent date separators to read Today/Yesterday.
+ - Added an autocomplete to the edit channel screen.
+ - Added support for pasting images on input text box.
+ - Emoji picker search now ignores the leading colon.
+ - Added support for emoji not requiring a whitespace to render.
+ - Added support for footer and footer_icon in attachments.
```
Comment (enahum) [inline CHANGELOG.md]:
```suggestion
 - Added support for footer and footer_icon in message attachments.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+ - Added a password type for interactive dialogs.
+ - Added support for introductory markdown paragraph in interactive dialogs.
+ - Added support for boolean elements in interactive dialogs.
+ - Removed the full screen "Refresh" button when opening a team or channel with no cached data when offline.
```
Comment (enahum) [inline CHANGELOG.md]: this wasn't removed it was improved with a better UX

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,59 @@
...
+ - If Mattermost doesn't have permission to the photo library because the user denied the first prompt, the app now prompts to give permission to send the user to the settings to manually change the permission.
+
+### Bug Fixes
+ - Fixed an issue where the notification badge could get out of sync when reading messages in another client.
+ - Fixed an issue where the notification badge number did not reset when opening a push notification.
+ - Fixed an issue where SafeArea insets were not working properly on new iPhone 11 models.
+ - Fixed an issue on iOS where long press on a system message in an archived channel locked up the app.
+ - Fixed an issue where tapping on a hashtag while replying to search results didn't open search correctly.
+ - Fixed an issue on iOS where the channel list panel was missing for a user when they were added to a new team by another user.
```
Comment (enahum) [inline CHANGELOG.md]:
```suggestion
 - Fixed an issue where the channel list panel was missing for a user when they were added to a new team by another user.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,59 @@
...
+ - Added support for introductory markdown paragraph in interactive dialogs.
+ - Added support for boolean elements in interactive dialogs.
+ - Removed the full screen "Refresh" button when opening a team or channel with no cached data when offline.
+ - If Mattermost doesn't have permission to the photo library because the user denied the first prompt, the app now prompts to give permission to send the user to the settings to manually change the permission.
+
+### Bug Fixes
+ - Fixed an issue where the notification badge could get out of sync when reading messages in another client.
+ - Fixed an issue where the notification badge number did not reset when opening a push notification.
+ - Fixed an issue where SafeArea insets were not working properly on new iPhone 11 models.
+ - Fixed an issue on iOS where long press on a system message in an archived channel locked up the app.
```
Comment (enahum) [inline CHANGELOG.md]:
```suggestion
 - Fixed an issue where long press on a system message in an archived channel locked up the app.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,59 @@
...
+ - Fixed an issue where once in a thread, pressing a channel link appeared to do nothing.
+ - Fixed an issue where file previews could scroll to the left until all files were out of view.
+ - Fixed an issue on iOS where user was unable to select an emoji from two rows on the bottom of the emoji picker.
+ - Fixed an issue where duplicate pinned posts displayed after editing pinned post from Pinned Posts screen.
+ - Fixed an issue where the reply arrow overlapped a posts's timestamp in some cases.
+ - Fixed an issue where post textbox did not clear after using a slash command.
+ - Fixed an issue where users were are not immediately removed from the mention auto-complete when those users were deactivated.
+ - Fixed an issue where returning to a channel from a thread view could trigger a long-press menu that couldn't be dismissed.
+ - Fixed an issue with a missing "(you)" suffix in the channel header.
+ - Fixed an issue where the Connected banner got stuck when connecting to the Web socket.
```
Comment (enahum) [inline CHANGELOG.md]:
```suggestion
 - Fixed an issue where the Connected banner got stuck after the WebSocket was connected.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,59 @@
...
+ - Fixed an issue where the text input area in Android Share extension did not use available space.
+ - Fixed an issue on iOS where Windows dark theme was not consistent when viewing an archived channel.
```
Comment (enahum) [inline CHANGELOG.md]:
```suggestion
 - Fixed an issue where Windows dark theme was not consistent when viewing an archived channel.
```

File: CHANGELOG.md (hunk 1 above)
Comment (enahum) [reply CHANGELOG.md]: moved a couple from improvements to highlight.

Also include `Added support for copy and paste protection managed configuration for Android`

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,60 @@
...
+### Highlights
+
+#### Sidebar and theming UI/UX improvements
+
+#### Ability to paste images on input text box
+
+#### Copy and paste protection managed configuration for Android
+
+### Improvements
+ - Added a confirmation dialog when posting a message with `at-channel`.
```
Comment (enahum) [inline CHANGELOG.md]:
```suggestion
 - Added a confirmation dialog when posting a message with `@channel` and `@all`.
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -9,7 +64,7 @@
  - iPhone 5s devices and later with iOS 11+ is required.
 
 ### Bug Fixes
- - Fixed issues causing the app to crash on some devices
+ - Fixed issues causing the app to crash on some devices.
```
Comment (enahum) [inline CHANGELOG.md]: If we cannot explicitly say which ones are these (that would make sense to the user), then I would say we should not add it

Comment (enahum) [reply CHANGELOG.md]: I believe we had more bug fixes, not only around crashes.

**Comment by amyblais** [comment]
@enahum Can you please help merge this so it's available for customers?

I can open a separate PR tomorrow if any last edits are needed.

**Pull request merged**